README.md

```diff
@@ -34,22 +35,39 @@ An example Excel file could look like:
         .                .                .
 ```
 
-### Usage
+In case of having a plural of the form:
 
+```text
+<plurals name="example_day">
+	<item quantity="one">1 day</item>
+	<item quantity="other">%1$d days</item>
+</plurals>
+```
+The script produces two rows in the sheet:
 
-`python i18n_xml_from_excel.py -f <input_excel_file> [-c]`
+```text
+--------------------------------------------------------------------
+|         	keys			 |  lang_code_1  |  lang_code_2  |  ...
```

Comment:
Use spaces instead of tabs so columns are aligned in code and after Mardown formatting is applied